src/main/resources/db/migration/V001__Init.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE message (
+  id UUID NOT NULL PRIMARY KEY,
+  message_id VARCHAR(256) NOT NULL,
+  service VARCHAR(256) NOT NULL,
+  created TIMESTAMP NOT NULL,
```

Comment:
What's the current convention of naming date columns? I like `created_at` or `created_date`, etc. (same with `printed`), but if we're already following the existing convention, then the change would make things inconsistent.

Reply:
+! for `xxx_at` (draft-store uses it)